gengetopt.spec

```diff
@@ -0,0 +1,39 @@
+### RPM external gengetopt 2.22.6
+Source: ftp://ftp.gnu.org/gnu/gengetopt/gengetopt-2.22.6.tar.gz
+Patch0: gengetopt-parallelbuild
+
+BuildRequires: autotools
+
+%define keep_archives true
```

Comment:
I was using some other package as a template, so I guess we can remove this line.